test-project-setup.sh

```diff
@@ -0,0 +1,53 @@
+#!/bin/bash
+# Copyright 2020 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+
+setup () {
+  # TEAM FOLDER must be in set in local environment
+  if [[ ! ${TEAM_FOLDER} ]]
```

Comment:
All the environment variables should be documented, in the tradition of bash script we could add a comment block near the top of the file naming the script, explaining what it does, and added the config options, but a separate README may be even better. Documenting a config option should include what it is, an example of what it looks like, and whether it's required vs. optional + default value.

Reply:
Added a README.  Do you still want a comment block? Do you have an example of a bash script that does this?